### PR TITLE
Add missing entries to IUnlock

### PR DESCRIPTION
**Pull request by NickCuso**
# Description

Adds a test to ensure the interface has all functions.

# Issues

Fixes #
Refs #6085 

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/contracts/interfaces/IUnlock.sol [hunk 1]
```diff
@@ -6,9 +6,22 @@ pragma solidity 0.5.16;
...
+   * We use deployed to keep track of deployments.
+   * This is required because both totalSales and yieldedDiscountTokens are 0 when initialized,
+   * which would be the same values when the lock is not set.
+   */
+  struct LockBalances
+  {
+    bool deployed;
+    uint totalSales; // This is in wei
+    uint yieldedDiscountTokens;
+  }
```
Comment (nfurfaro) [inline smart-contracts/contracts/interfaces/IUnlock.sol]: Not sure about this, moving storage to the interface?

Comment (NickCuso) [reply smart-contracts/contracts/interfaces/IUnlock.sol]: It's just the struct itself - not any data.  We will need this in the interface once we switch to abiv2 or Solidity 0.6 and want to start exposing structures publicly.

Comment (nfurfaro) [reply smart-contracts/contracts/interfaces/IUnlock.sol]: copy that.

Comment (NickCuso) [reply smart-contracts/contracts/interfaces/IUnlock.sol]: But for now we could move it back...

Comment (NickCuso) [reply smart-contracts/contracts/interfaces/IUnlock.sol]: Yea I'm going to move it back for now. We can revisit when we need it, but for now this is the first instance of info only available in the interface...

Comment (nfurfaro) [reply smart-contracts/contracts/interfaces/IUnlock.sol]: Allright. I'm fine either way here if we'll need it soon enough.

**Pull request merged**